README.md

```diff
@@ -100,6 +100,52 @@ You are free to implement the genome classes however you want, and using whateer
 
 When you have implemented the two (or more) classes, describe the complexity of each operation as a function of the genome size (at the time of the operation), and the size of the TE involved (and when copying, the offset you are copying). Put the description here:
 
-**FIXME: OPERATION COMPLEXITY**
+# Awnser:
+
+*ListGenome:*
+
+init = O(n), creating a list of length n.
+
+insert = O(te + n), creating a list of size te + inserting this list + (correcting).. probably something like te^2 * n i practise. 
```

Comment:
There is no quadratic to it, but you run through all existing TEs. So, if the genome has length $n$, there are $k$ TEs, and the new has length $m$, the running time is $O(n+k+m)$.  Since $k$ must be less than $n$--we don't have more active TEs than we have genome, this is also $O(n+m)$.